pysatNASA/instruments/__init__.py

```diff
@@ -2,7 +2,7 @@
            'de2_lang', 'de2_nacs', 'de2_rpa', 'de2_wats',
            'formosat1_ivm',
            'icon_euv', 'icon_fuv', 'icon_ivm', 'icon_mighti',
-           'iss_fpmu', 'omni_hro', 'sport_ivm',
+           'iss_fpmu', 'omni_hro', 'ses1_gold', 'sport_ivm',
```

Comment:
```suggestion
           'iss_fpmu', 'omni_hro', 'ses14_gold', 'sport_ivm',
```

I misread the NASA page.  Here's the sc info: https://space.skyrocket.de/doc_sdat/ses-14.htm